projects/sport-data-valley/seeds/protocols/mmm.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../protocol_helper'
+require_relative '../questionnaires/mmm_questionnaire'
+
+MMM_PROTOCOL_NAME = 'move_mood_motivation'
+MMM_NOTIFICATION_NAME = 'base-platform-subscription-move-mood-motivation'
```

Comment:
Where is this used?

Reply:
It should be used instead of `bp_name`. Nice find